### DISCUSSION TITLE
Display "fails" in graph

### DISCUSSION THREAD

**Discussion by OPTips** [alexjustesen/speedtest-tracker#1721] [Ideas]
Hi,

I do not know if this is already possible or even the question has been already asked, but here what I'd like to see in my dashboard.

When a test fails, it is not displayed in graphs. For instance: 

![image](https://github.com/user-attachments/assets/07343b11-deca-47f9-8631-ea0131f0d043)

Here you can see that my internet connection was down for a while.

![image](https://github.com/user-attachments/assets/c123763a-fff1-4714-87b7-e08e31a8d94d)

But here you can see there is a jump between 19th of September and 22nd of September. It would be nice if I can see it so I can view that there was no internet.

What do you think?

Have fun

**Comment by svenvg93** [comment]
Hi @OlivierProTips, 

There are similar request open, and its definitely on the list to make this visible in the graphs. No ETA yet, there first need to be some other fixes implemented before this can be done :) 

Thanks for the suggestion! 

Sven

**Reply by OPTips** [reply]
Thanks a lot. This is good to hear. :)
